Instructor-Guide--Assignments--Summary.md

```diff
@@ -43,11 +45,32 @@ To the right of the grade distribution chart is a set of statistics that give a
+#### Criteria Distribution
+
+Below the grade distribution chart, you can further analyze the distribution of grades using the criteria distribution graph. This graph shows the distribution of marks given for each associated criterion.
+
+![Criterion Grade Distribution Graph](images/criteria-summary-stats-grade-distribution.png)
+
+Each criterion corresponds to a colour shown in the labels just above the graph. By default, all grade data for criterions are hidden. In order to reveal the grade distribution for a specific criterion(s), simply click on the labels of all the criterion you wish to view and compare.
+
+To help you get a quick overview of student performance for each criteria, next to the criteria distribution graph is a table that shows each criterion along with the average grade received for that criterion. Clicking on the drop down arrow next to each criterion will reveal additional summary statistics for that particular criterion.
+
+![Criterion Grade Distribution Table](images/criteria-summary-stats-table.png)
+
+The additional criterion statistics currently shown are:
+
+- **Average:** The average point grade of submissions for the given criterion (including submissions which recieved a zero) out of the maximum possible mark for that criterion.
+- **Median:** The median grade of submissions for the given criterion (including submissions which recieved a zero) out of the maximum possible mark for that criterion.
+- **Standard deviation:** The point grade standard deviation of submissions for the given criterion (including submissions which recieved a zero). In brackets next to this statistic is the standard deviation of submissions for the given criterion as a percentage grade.
+- **Number of zeros:** How many graded submissions recieved a grade of zero on this particular criterion out of the expected total number of submissions.
+
+#### Grader Distribution
+
+Finally, at the bottom of the view, below the criteria distribution chart, you can see another distribution of grades for an assignment. What differentiates this grade distribution chart from the normal chart at the top of the page however, is that this chart breaks down the normal grade distribution graph in order to also show the distribution of marks given by each grader.
 
 ![Assignment Grader Distribution Graph](images/summary-stats-grader-distribution.png)
 
-Each grader corresponds to a colour shown in the labels just above the graph. Each label also shows how many submissions each grader has completed marking out of the number they were assigned.
+As with the criterion distribution, each grader corresponds to a colour shown in the labels just above the graph. Each label also shows how many submissions each grader has completed marking out of the number they were assigned.
```

Comment:
You use `criteria distribution` elsewhere. Please change this one to be consistent with the others